src/Response.php

```diff
@@ -107,7 +107,7 @@ public function write($data)
         return $flushed;
     }
 
-    public function end($data = null)
+    public function end($data = null, $keepAlive = false)
```

Comment:
Yeah it could be neater for the http framework to handle closing connections rather than leaving each request handler do it. 

The Request object could process the received headers according to http://tools.ietf.org/html/rfc7230#section-6.3 to know when to close a connection. In addition the writeHead function would need to check for presence of a "Connection: Close" header in the response, to close the connection when present.  

This would seem to mimic the behaviour of PHP scripts running under Apache.